frontend/helpers/generateFakeCollateral.js

```diff
@@ -4,13 +4,18 @@ import COLLATERALS from 'auctions-core/src/constants/COLLATERALS';
 
 export const generateFakeCollaterals = function () {
     const collaterals = [];
+
     for (const collateralObject of Object.values(COLLATERALS)) {
         collaterals.push({
             ilk: collateralObject.ilk,
             symbol: collateralObject.symbol,
-            marketUnitPrice: parseFloat(faker.finance.amount()),
-            secondsBetweenPriceDrops: faker.datatype.number(120),
+            marketUnitPrice: new BigNumber(faker.finance.amount()),
+            secondsBetweenPriceDrops: new BigNumber(faker.datatype.number(120)),
             priceDropRatio: new BigNumber(faker.datatype.number({ min: 0.5, max: 1, precision: 0.0001 })),
+            uniswap: {
+                type: 'token',
+                route: faker.finance.ethereumAddress(),
+            },
```

Comment:
Ok, I got the confusion now:
- Token address has nothing to do with uniswap or the route
- Even if it does, you're reusing collateral object structure for something completely different, which is confusing
- I suggest you just create a new key like `tokenAddress` where you store this information
```suggestion
            tokenAddress: faker.finance.ethereumAddress(),
```

Reply:
I see. I let myself get confused by the "token" type and thought they were the same thing. Sorry for the confusion 😅